### PR TITLE
Add truffle plugin initial structure

### PR DESCRIPTION
**Pull request by mariano-aguero**
No issues related

### CODE REVIEW

**Comment by pablofullana** [comment]
@mariano-aguero there are some missing empty lines at EOF

File: src/helpers.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const printHelpMessage = () => {
+  console.log(
+    [
+      "Usage: truffle run solhint [options]",
+      "",
+      "Options",
+      "-v, --version                          Show package and Solhint version information",
+      "-t, --target                           Path to the contracts. Default path is 'contracts'"
+    ].join("\r\n")
```
Comment (fvictorio) [inline src/helpers.js]: https://nodejs.org/api/os.html#os_os_eol

File: src/solhint.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      }
+    );
+
+    solhint.on("stdout", data => {
+      console.log(data.toString());
+      resolve(data.toString());
+    });
+
+    solhint.on("stderr", data => {
+      console.log(data.toString());
```
Comment (fvictorio) [inline src/solhint.js]: Maybe we should use `console.error` here.

Comment (mariano-aguero) [reply src/solhint.js]: Done

File: src/helpers.js (hunk 1 above)
Comment (mariano-aguero) [reply src/helpers.js]: Done

**Comment by mariano-aguero** [comment]
> @mariano-aguero there are some missing empty lines at EOF

We are using prettier, and currently prettier ensures that the file ends with a newline. 
I think we should leave it like that, Regards!

**Comment by fvictorio** [comment]
As it is, this doesn't work, at least for me.

The first problem is that `.solhint.json` is not included in the package. You can check this by running `npm pack` and inspecting the generated tar file. Adding:

```
  "files": [
    "src",
    ".solhint.json",
    "truffle-plugin.json"
  ],
```

to the `package.json` seems to fix this.

The second problem is that the `-c` option doesn't seem to be working in `solhint`.

**Comment by mariano-aguero** [comment]
> As it is, this doesn't work, at least for me.
> 
> The first problem is that `.solhint.json` is not included in the package. You can check this by running `npm pack` and inspecting the generated tar file. Adding:
> 
> ```
>   "files": [
>     "src",
>     ".solhint.json",
>     "truffle-plugin.json"
>   ],
> ```
> 
> to the `package.json` seems to fix this.

Solved the first issue

**Comment by mariano-aguero** [comment]
For the second issue, connects with the issues:
- https://github.com/protofire/solhint/pull/126
- https://github.com/protofire/solhint/pull/127

**Review by fvictorio — APPROVED** [review]

**Pull request merged**